test/integration/targets/vmware_inventory/runme.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+# Get Current test directory
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+contrib_dir=$(pwd)/../../../../contrib/inventory
+cd "${contrib_dir}"
+
+if [ ! -f ./vmware_inventory.py ]; then
+    echo "DEBUG: Unable to find vmware_inventory.py inside ${contrib_dir}"
+    exit 1
+fi
+
+echo "DEBUG: using ${contrib_dir}"
+
+export ANSIBLE_CONFIG=ansible.cfg
+export vcenter_host="${vcenter_host:-0.0.0.0}"
+export VMWARE_SERVER="${vcenter_host}"
+export VMWARE_USERNAME="${VMWARE_USERNAME:-user}"
+export VMWARE_PASSWORD="${VMWARE_PASSWORD:-pass}"
+
+VMWARE_CONFIG=${contrib_dir}/vmware_inventory.ini
+
+# Remove default inventory config file
+if [ -f "${VMWARE_CONFIG}" ];
+then
+    echo "DEBUG: Creating backup of ${VMWARE_CONFIG}"
+    cp "${VMWARE_CONFIG}" "${VMWARE_CONFIG}.bk"
+fi
+
+cat > "${VMWARE_CONFIG}" <<VMWARE_INI
+[vmware]
+server=${VMWARE_SERVER}
+username=${VMWARE_USERNAME}
+password=${VMWARE_PASSWORD}
+validate_certs=False
+VMWARE_INI
+
+function cleanup {
+    # Revert back to previous one
+    if [ -f "${VMWARE_CONFIG}.bk" ]; then
+        echo "DEBUG: Cleanup"
+        mv "${VMWARE_CONFIG}.bk" "${VMWARE_CONFIG}"
+    fi
+}
+
+trap cleanup INT TERM EXIT
```

Comment:
Register the trap before backing up the file. Otherwise a failure between the two steps won't restore the original config.

Reply:
OK.